app/models/ext_management_system.rb

```diff
@@ -99,6 +102,22 @@ def hostname_format_valid?
     errors.add(:hostname, _("format is invalid."))
   end
 
+  # validation - Zone cannot be changed when enabled == false
+  def zone_unchanged_when_paused?
```

Comment:
something about the unchanged when is confusing in the method name...trying to come up with something a little clearer, but I'm not sure what this does.

Reply:
I was about to suggest naming this `validate_ems_disabled_when_zone_changed?` and I realized it;s the exact same wording I gave below, which leads me to wonder what the difference is between this and the next method.

Reply:
difference is, that this validation controls that EMS.zone cannot be changed when provider is paused.

The second validation ensures that EMS in normal state (unpaused) cannot be in invisible zone (currently maintenance zone only)